packages/jest-config/src/Defaults.ts

```diff
@@ -19,11 +20,14 @@ const defaultOptions: Config.DefaultOptions = {
   cache: true,
   cacheDirectory: getCacheDirectory(),
   changedFilesWithAncestor: false,
+  ci: isCI,
   clearMocks: false,
   collectCoverage: false,
   coveragePathIgnorePatterns: [NODE_MODULES_REGEXP],
   coverageProvider: 'babel',
   coverageReporters: ['json', 'text', 'lcov', 'clover'],
+  detectLeaks: false,
+  detectOpenHandles: false,
```

Comment:
I'm fine with removing the defaults from args, but we left some flags like these out of config on purpose. Don't we want to do that anymore?

Reply:
Oh, i didn't know that. I can update my PR in this way. Let me know what fits the project needs.

Reply:
Yes, we wanna keep that

Reply:
@SimenB ICMI: https://github.com/facebook/jest/pull/6747

Reply:
yeah, I don't want that one 😀

Reply:
Let's close it then :)

Reply:
Ok, to summarize, i need to update my PR to keep the `detectOpenHandles` default value in CLI options. Any other?

Reply:
wait, I don't think it matters if the default is here? As longs as it's `false` I mean. We just don't wanna read it from the user's config file.

@ghostd it's all the new ones added here - we want people to be able to set them from the CLI but not from config (at least not yet, we might change that)

Reply:
`jest-cli` already depends on `jest-config`, so does it make sense to use defaultOptions to init the CLI options object. Hence we'll have the default values in jest-config and all parameters having a default value in the yarg object will be not overrided by the user config.